### PR TITLE
[skip ci] Added missing docker commands and incorporated more edits

### PR DESCRIPTION
**Pull request by npakrasi**
Docker Commands 
- Added unsupported docker commands that were missing from Cheng's review.
- Added note about not supporting Swarm from Loc.
- Added network connect comment from Hasan
- Deleted an extra Yes.

Security Reference: 
- Added --ops-user and --ops-password options
Fixes # 3353, 3404

### CODE REVIEW

**Comment by npakrasi** [comment]
LGTM

File: doc/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -1,38 +1,50 @@
...
+|import|[Import the contents from a tarball to create a filesystem image](https://docs.docker.com/engine/reference/commandline/import/)|No|
 |info|[Docker system information](https://docs.docker.com/engine/reference/commandline/info/)|Yes, docker-specific data, basic capacity information, list of configured volume stores, virtual container host information. Does not reveal vSphere datastore paths that might contain sensitive vSphere information.|
 |inspect|[Inspect a container or image](https://docs.docker.com/engine/reference/commandline/inspect/)|Yes. Includes information about the container network.|
 |kill|[Kill a running container](https://docs.docker.com/engine/reference/commandline/kill/)|Yes. Docker must wait for the container to shut down.|
-|login| [Log into a Docker registry](https://docs.docker.com/engine/reference/commandline/login/)|Yes|
+|load|[Load an image from a tar archive or STDIN](https://docs.docker.com/engine/reference/commandline/load/)|No|
+|login| [Log into a registry](https://docs.docker.com/engine/reference/commandline/login/)|Yes|
+|logout| [Log out from a registry](https://docs.docker.com/engine/reference/commandline/logout/)|No|
```
Comment (npakrasi) [inline doc/user_doc/vic_app_dev/container_operations.md]: Changed to Yes.

File: doc/user_doc/vic_app_dev/container_operations.md [hunk 2]
```diff
@@ -43,7 +55,7 @@ For more information about network operations, see [Network Port Use Cases](netw
 
 | **Network** | **Docker Reference** | **Supported** |
 | --- | --- | --- |
-|Network connect|[Connect to a network](https://docs.docker.com/engine/reference/commandline/network_connect/)|Yes|
+|Network connect|[Connect to a network](https://docs.docker.com/engine/reference/commandline/network_connect/)|Yes, not supported for running containers|
```
Comment (npakrasi) [inline doc/user_doc/vic_app_dev/container_operations.md]: Edited accordingly. Thanks!

**Comment by sflxn** [comment]
lgtm

**Comment by npakrasi** [comment]
.

**Pull request merged**